stable/elasticsearch-exporter/templates/servicemonitor.yaml

```diff
@@ -0,0 +1,23 @@
+{{ if .Values.serviceMonitor.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: {{ include "elasticsearch-exporter.fullname" . }}
+  {{- if .Values.serviceMonitor.namespace }}
+  namespace: {{ .Values.serviceMonitor.namespace }}
+  {{- end }}
+  labels:
+    {{- range $key, $value := .Values.serviceMonitor.selector }}
```

Comment:
I recommend changing this from `serviceMonitor.selector` to `serviceMonitor.labels`.
The label selector is defined in the operator config.

Reply:
done